src/commands/blocking_commander.h

```diff
@@ -44,7 +44,6 @@ class BlockingCommander : public Commander,
   // in other words, returning true indicates ending the blocking
   virtual bool OnBlockingWrite() = 0;
 
-  bool IsBlocking() const override { return true; }
```

Comment:
Now we do not need to check if it is blocking or not?

Reply:
Yes, we don't migrate the connection once it has the running command to avoid data race between the new and old worker.